devpack/src/main/java/io/neow3j/devpack/constants/StackItemType.java

```diff
@@ -0,0 +1,14 @@
+package io.neow3j.devpack.constants;
+
+public class StackItemType {
+    public static final byte ANY = 0x00;
```

Comment:
For example, like this:

```java
public static final byte ANY = io.neow3j.types.StackItemType.ANY.byteValue();
```

Reply:
@csmuller 👆🏼

Reply:
Basically still relying on the `core` module and it's values, all while abstracting the types that are necessary and need to be disclosed to the developer.

I usually just dislike copy pasting the same values to reduce future maintenance load if something were to change. However, I know that it's likely that none of these constant values will change, so I'm not sure it's necessary. Just thinking out loud.

Reply:
Good point. I will have to check if this solution will lead to the same warnings that the dev experiences now. If not, it's a good way to make sure we have one central point where we maintain this enums and don't have real duplication.

Reply:
It doesn't work because when using `StackItemType` in an annotation (example below), then the value of the static variable (e.g., of `StackItemType.INTEGER`) must be a constant expression and cannot be a method invocation as in `byteValue`.
```
 @Instruction(opcode = OpCode.CONVERT, operand = StackItemType.INTEGER)
 ```